tests/test_escape_attr.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from nixpkgs_review.utils import escape_attr
+
+
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        ("haskellPackages.if", 'haskellPackages."if"'),
+        ("haskellPackages.if.doc", 'haskellPackages."if"."doc"'),
+    ],
+)
+def test_escape_attr(test_input: str, expected: str) -> None:
+    assert escape_attr(test_input) == expected
```

Comment:
My brain has an easier timer parsing this and it is less code:

```suggestion
    assert escape_attr("haskellPackages.if") == 'haskellPackages."if"'
    assert escape_attr("haskellPackages.if.doc") == 'haskellPackages."if"."doc"'
```